api/specs.proto

```diff
@@ -258,6 +258,12 @@ message ContainerSpec {
 	// task will exit and a new task will be rescheduled elsewhere. A container
 	// is considered unhealthy after `Retries` number of consecutive failures.
 	HealthConfig healthcheck = 16;
+
+	message SecurityConfig {
+	    string userns = 1;
+	    string credential_spec = 2 [(gogoproto.customname) = "CredentialSpec"];
```

Comment:
Yes it's windows specific.  I believe it specifies a path to a "credential blob" (https://github.com/docker/docker/pull/23389).  This PR was supposed to be about security configuration and not specific to security profiles (although full disclosure I'm not sure what exactly security profiles mean as related to docker).  Since they are authentication related and point to what seems like an authentication blob I thought it would be classified as "security".

Based upon the latest comment https://github.com/docker/docker/pull/30894#issuecomment-279047540 this whole thing might be moot.